at_sync_ui_flutter/example/ios/Podfile.lock

```diff
@@ -113,17 +126,20 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   at_backupkey_flutter: 2fc3d01138175e41bce8b574387a47544c53e01b
+  at_file_saver: c0e052c72d8c0296318bd70f2ae7f510887014ce
   at_onboarding_flutter: e8219b6d0bfb236d3837ec3528871aebdcc56e8d
   biometric_storage: 1400f1382af3a4cc2bf05340e13c3d8de873ceb9
   DKImagePickerController: b5eb7f7a388e4643264105d648d01f727110fc3d
   DKPhotoGallery: fdfad5125a9fdda9cc57df834d49df790dbb4179
   file_picker: 817ab1d8cd2da9d2da412a417162deee3500fc95
-  file_saver: 503e386464dbe118f630e17b4c2e1190fa0cf808
   Flutter: 50d75fe2f02b26cc09d224853bb45737f8b3214a
   flutter_keychain: 01aabf894ffe8b01adfda1d9df21c210c1b4b452
   flutter_qr_reader: d930dde3b2cfe2b3d0bb7d66e5ff3e514300a5e5
```

Comment:
here too. Try removing the `.lock` file and try regenerating or you can ignore this file in `.gitignore`

Reply:
Lock files should be ignored in libraries, that way dependencies can roll forward without us having to manually republish the lockfile.

Reply:
@yahu1031 at_sync_ui is using older version of at_onboarding_flutter.
that's why it still has flutter_qr_reader as dependency.
This will be removed once we deploy new version of at_onboarding_flutter.